core/vm/opcodes.go

```diff
@@ -101,6 +101,7 @@ const (
 	NUMBER
 	DIFFICULTY
 	GASLIMIT
+	SELFBALANCE = 0x47
```

Comment:
Any particular reason for explicitly setting `0x47`? As far as I see it that's the next in line anyway

Reply:
Hah, then you're seeing wrong :) Next available slot is `0x46`, which is reserved for `CHAINID`. 

But really, that shows the danger of using `iota` for this. `iota` should IMO only be used for internals, where it's fine to drop something and everything magically rearranges itself. I don't like that we use it like this, where instead we should explicitly define every opcode.